app/models/manageiq/providers/kubernetes/container_manager/refresh_parser.rb

```diff
@@ -692,6 +686,8 @@ def parse_container_image(image, imageID)
         if stored_container_image_registry.nil?
           @data_index.store_path(
             :container_image_registry, :by_host_and_port, host_port, container_image_registry)
+          @data[:container_image_registries] ||= []
+          @data[:container_image_registries] << container_image_registry
```

Comment:
@enoodle it depends on where you'll put that line. Can you update the PR so we'll see how it looks like.